### ISSUE TITLE
Compare to other tools

### ISSUE THREAD

**Issue by bonndan** [myplaceonline/posixcube#4]
Very interesting project. I thought you might add a few lines in the project description comparing posixcube to ansible, salt, chef, puppet etc. What are the pros and cons?

**Comment by kevgrig** [comment]
Good idea. I'll have to research those more to make a fair comparison...

**Comment by kevgrig** [comment]
Another tool: https://pressly.github.io/sup/

**Comment by kevgrig** [comment]
And another: http://www.fabfile.org/

**Comment by kevgrig** [comment]
I like the idea of using some real-world example/use case for comparison, such as the one discussed here: https://news.ycombinator.com/item?id=13386294

**Comment by aero** [comment]
Similar tool: https://github.com/bltavares/kickstart

**Comment by aero** [comment]
https://github.com/pch/welder

Show HN: Welder – set up your Linux server with plain shell scripts
 https://news.ycombinator.com/item?id=14522520

**Comment by kevgrig** [comment]
This is good to keep a running list of alternative projects. I haven't reviewed them yet and maybe some support this, but one key requirement that I have (which posixcube supports) is to allow encrypted key=value files (which are uploaded, decrypted, and then deleted).

**Comment by aero** [comment]
http://www.bashbooster.net/


BashBooster – simple provisioning, bash only, requires nothing
https://news.ycombinator.com/item?id=8344070

**Comment by aero** [comment]
Bashible: An Ansible-inspired deployment/automation tool written in Bash DSL 
https://news.ycombinator.com/item?id=22669565

https://github.com/mig1984/bashible
